backend/app/controllers/spree/admin/option_types_controller.rb

```diff
@@ -21,7 +21,7 @@ def location_after_save
       end
 
       def load_product
-        @product = Spree::Product.find_by_param!(params[:product_id])
+        @product = Spree::Product.find_by!(param: params[:product_id])
```

Comment:
I think the best thing for me, with my limited knowledge of what is happening here, is to revert the code to what it was before.